src/translations/fr.json

```diff
@@ -209,6 +209,11 @@
     "ConverterStationId2": "ID station conversion 2",
     "NominalV": "Tension nominale (kV)",
     "HvdcLineId": "ID ligne HVDC",
+    "OprFromCS1toCS2": "LIMITE DE FONCTIONNEMENT (COTE_1 VERS COTE_2) (MW)",
+    "OprFromCS2toCS1": "LIMITE DE FONCTIONNEMENT (COTE_2 VERS COTE_1) (MW)",
+    "AcEmulation": "EMULATION AC",
+    "K": "GAIN (MW/deg)",
+    "P0": "P0 (COTE_1 VERS COTE_2) (MW)",
```

Comment:
Do not write translations only with uppercases, even if the name of the column in the table is displayed in uppercases. When configuring displayed columns (by clicking on 'Edit columns' button), the names of the columns is not converted to uppercases.